### PR TITLE
Add support for reading external Markdown file for schema documentation, then attaching it

### PR DESCRIPTION
**Pull request by hlship**
Feedback on the naming of the various functions ... I'm vacillating between `attach-documentation` and `attach-descriptions`.

### CODE REVIEW

**Comment by hlship** [comment]
I’m now l among towards inject-descriptions. I’m also thinking of adding an inject-resolvers

**Comment by annapawlicka** [comment]
I'll merge it once the conflicts are resolved.

File: CHANGES.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ See `com.walmartlabs.lacinia.parser.docs/parse-docs` and
 Lacinia now supports the `:roots` key in the input schema, which makes
 it possible to define query, mutation, or subscription operations
 in terms of the fields of an explicitly named object in the schema.
-This aligns Lacinia better with other implementations of GraphQL.
+This aligns Lacinia better withu other implementations of GraphQL.
```
Comment (annapawlicka) [inline CHANGES.md]: typo `withu`

**Comment by hlship** [comment]
Conflicts resolved, and shuffled some code into internal-utils

**Pull request merged**